src/openfl/display/Stage.hx

```diff
@@ -918,8 +912,15 @@ class Stage extends DisplayObjectContainer implements IModule {
 		
 		if (this.window == null || this.window != window) return;
 		
-		//__broadcastEvent (new Event (Event.ACTIVATE));
-		
+		__handleFullScreenRestore ();
+	}
+	
+	inline function __handleFullScreenRestore() {
```

Comment:
Fields in openFL are sorted alphabetically. Please move this method where it alphabetically fits

Reply:
oh come on...

Reply:
As discussed, I don't think this is a good practice and we shouldn't sacrifice logical grouping on the altar of weird code style requirements. To me it makes much more sense to have this function close to its usages so I would rather not change it.

BTW, the upstream codebase doesn't have this problem because this code is just copy-pasted :)